src/main/java/it/gov/pagopa/payhub/activities/dto/DebtPositionTypeOrgDTO.java

```diff
@@ -12,10 +12,9 @@
 @Builder(toBuilder = true)
 @NoArgsConstructor
 @AllArgsConstructor
-public class OrganizationInstallmentTypeDTO implements Serializable {
+public class DebtPositionTypeOrgDTO implements Serializable {
 
-
-    private Long installmentTypeId;
+    private Long debtPositionTypeId;
```

Comment:
this entity should have also a debtPositionTypeId, which actually doesn't exists!
```suggestion
    private Long debtPositionTypeOrgId;
```